<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>434</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>574</width>
    <height>434</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>574</width>
    <height>434</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>621</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>0</y>
      <width>621</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/background.jpg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton{
background-color: rgb(202, 87, 242);
font: 16pt &quot;Cooper Black&quot;;
border-radius: 15px;
color: black; 
border: 2px solid rgb(202, 87, 242);
}

.QPushButton:hover {
  background-color: rgb(255, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton{
background-color: rgb(202, 87, 242);
font: 8pt &quot;Cooper Black&quot;;
border-radius: 15px;
color: black; 
border: 2px solid rgb(202, 87, 242);
}

.QPushButton:hover {
  background-color: rgb(255, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Performance</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>391</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Start Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stop Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Duration (second)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Q1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Q2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Q3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Q4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Q5</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
